packages/familie-header/src/søk/Søk.less

```diff
@@ -3,10 +3,12 @@
 @inactiveColor: @navGra60;
 @activeColor: @navOransjeLighten80;
 @focusColor: @orangeFocus;
+@fontColor: #d7dae0;
 @searchWidth: 18rem;
 
 .søk_container {
-  position: relative;
+    position: relative;
+    font-family: 'Source Sans Pro', Arial, Helvetica, sans-serif;
```

Comment:
Ingen peiling på slike ting, men burde dette egentligen være noe som ligger på en høyere nivå?

Reply:
Nja, greit nok at den ligger på denne komponenten. Bør ikke settes "globalt" for komponenter, iallefall.

Reply:
Hvorfor kan ikke dette settes av konsumenten? I ba-sak tror jeg vi har satt font for hele appen i index.less.

Reply:
Forsåvidt greit poeng hva gjelder font-er. Jeg fjerner denne og oppretter PR.